.github/workflows/master.yml

```diff
@@ -37,6 +37,84 @@ jobs:
         env:
           OUT: ${{ steps.get-tag.outputs.sha }}
 
+  build_native_linux:
+    runs-on: ubuntu-20.04
+    timeout-minutes: 60
+    name: Build Linux Native backend for Dev image
+    container:
+      image: cubejs/rust-cross:x86_64-unknown-linux-gnu-15082024-python-3.9
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Install Rust
+        uses: actions-rust-lang/setup-rust-toolchain@v1
+        with:
+          toolchain: nightly-2024-07-15
+          # override: true # this is by default on
+          rustflags: ""
+          components: rustfmt
+          target: x86_64-unknown-linux-gnu
+          cache: false
+      - uses: Swatinem/rust-cache@v2
+        with:
+          workspaces: ./rust/cubesql -> target
+          key: cubesql-x86_64-unknown-linux-gnu
+          shared-key: cubesql-x86_64-unknown-linux-gnu
+      - name: Install Node.js 18
+        uses: actions/setup-node@v4
+        with:
+          node-version: 18
+      - name: Install Yarn
+        run: npm install -g yarn
+      - name: Set Yarn version
+        run: yarn policies set-version v1.22.19
+      - name: Get yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo "dir=$(yarn cache dir)" >> "$GITHUB_OUTPUT"
+      - name: Restore yarn cache
+        uses: actions/cache@v4
+        with:
+          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+      - name: Yarn install
```

Comment:
We dont need to install all packages, compiling tsc
let's use what we use in the publish

```
      - name: Set Yarn version
        run: yarn policies set-version v1.22.19
      - name: Install cargo-cp-artifact
        run: npm install -g cargo-cp-artifact@0.1

```